dashboard/app/views/pd/pre_workshop_survey/new.html.haml

```diff
@@ -0,0 +1,17 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+  %script{src: minifiable_asset_path('js/pd/pre_workshop_survey/new.js'), data: @script_data}
+
+%h1
+  Pre-survey for your upcoming
```

Comment:
Stylistic, but you can do `= "Pre-survey for your upcoming #{@pd_enrollment.workshop.course} workshop"`